lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -440,4 +440,10 @@ defmodule LivebookWeb.SessionLive.CellComponent do
   end
 
   defp evaluated_label(_time_ms), do: nil
+
+  defp html_input_type(:password), do: "password"
+  defp html_input_type(:number), do: "number"
+  defp html_input_type(:url), do: "url"
```

Comment:
We validate the URL on our own and the url input doesn't provide anything except for validation, so I think it's better to stick to text there. For number I'm not sure, but perhaps the up/down arrows are useful sometime so let's try this out.

Reply:
I think the number one is quite nice too because it rejects invalid key presses - so you can't type 'a' and then get the validation error. Happy to change url back, the only other thing I'm aware of that it does is it'll often tweak virtual keyboards, but that probably isn't very compelling here.

Reply:
I've removed the case for `url`